sale_partner_incoterm/models/sale_order.py

```diff
@@ -14,11 +14,9 @@ class SaleOrder(models.Model):
 
     @api.onchange("partner_id")
     def onchange_partner_id(self):
-        res = super().onchange_partner_id()
         self.update(
             {
                 "incoterm": self.partner_id.sale_incoterm_id.id,
                 "incoterm_address_id": self.partner_id.sale_incoterm_address_id.id,
             }
         )
-        return res
```

Comment:
Why are you no longer calling super?

AFAICS, then this won't be executed anymore, and that'd be wrong: https://github.com/odoo/odoo//blob/b72dc83b6050c6b662d6d85ebb94cdb9709aa433/addons/purchase/models/purchase.py#L294